pages/docs/reference/mpp-dsl-reference.md

```diff
@@ -270,6 +268,99 @@ kotlin {
 
 </div>
 
+For binaries configuration, the following parameters are available:
+
+|**Name**|**Description**| 
+| --- | --- |
+|`compilation`|The compilation from which the binary is built. By default, `test` binaries are based on the `test` compilation while other binaries - on the `main` compilation.|
+|`linkerOpts`|Options passed to a system linker during binary building.|
+|`baseName`|Base name for the output file.|
```

Comment:
Don't you want to add information about the difference between the base and final output file names?